google/cloud/storage/internal/object_acl_requests_test.cc

```diff
@@ -0,0 +1,52 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/storage/internal/object_acl_requests.h"
+#include <gmock/gmock.h>
+
+namespace google {
+namespace cloud {
+namespace storage {
+inline namespace STORAGE_CLIENT_NS {
+namespace internal {
+namespace {
+
+TEST(ObjectAclRequestTest, Simple) {
+  ObjectAclRequest request("my-bucket", "my-object", "user-test-user");
+  EXPECT_EQ("my-bucket", request.bucket_name());
+  EXPECT_EQ("my-object", request.object_name());
+  EXPECT_EQ("user-test-user", request.entity());
+}
+
+using ::testing::HasSubstr;
```

Comment:
Nit: Would you mind putting this at the top of this unnamed namespace?  I'd like to keep using directives and aliases near the top of the test files (I have a CL almost finished that cleans a lot of that up).